### PR TITLE
[Service] Added NAT entry into CONTAINER_FEATURE. Fixes #4247.

### LINKED ISSUE #4247: [supervisor-proc-exit-listener] Error message in syslog if feature is not supported
**Description**
Error message in syslog appears if feature is not supported
After changes, made in https://github.com/Azure/sonic-buildimage/pull/4208 for nat container, error messages appears in syslog.

**Steps to reproduce the issue:**
Run crm test on t0 topology:
`ansible-playbook test_sonic.yml -i inventory --limit <testbed> -e testbed_name=<testbed> -e testbed_type=t0 -e testcase_name=crm`
**After this changes also failed:** 
dhcp_relay
vxlan_decap
acl
interfaces
everflow_testbed
lldp
continuous_link_flap
pfc_wd
neighbour_mac_noptf
restart_swss
**Describe the results you received:**
Test fails due error in syslog: 
`ERR nat#supervisor-proc-exit-listener: Unable to retrieve features for container 'nat'. Exiting...`
**Describe the results you expected:**
- all affected test cases passed 
- If feature is not supported, has to be implemented some new logic (INFO of DEBUG message in syslog, not ERROR)

**Additional information you deem important (e.g. issue happens only occasionally):**
For nat service option autorestart was enabled in https://github.com/Azure/sonic-buildimage/pull/4208. Error message appears here https://github.com/Azure/sonic-buildimage/blob/master/files/scripts/supervisor-proc-exit-listener lines 64-66.

    **Output of `show version`:**

    ```
    SONiC Software Version: SONiC.HEAD.591-dirty-20200309.044431
    Distribution: Debian 9.12
    Kernel: 4.9.0-11-2-amd64
    Build commit: 590caaf5
    Build date: Mon Mar  9 06:33:21 UTC 2020
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

### PR DESCRIPTION
**Pull request by akokhan**
Signed-off-by: Andriy Kokhan <akokhan@barefootnetworks.com>

**- What I did**
Added NAT entry into CONTAINER_FEATURE in config_db.json template

### CODE REVIEW

**Comment by akokhan** [comment]
@jleveque , @SavchukRomanLv , please have a look

File: files/build_templates/init_cfg.json.j2 [hunk 1]
```diff
@@ -26,7 +26,7 @@
     },
     "CONTAINER_FEATURE": {
 {%- for container in ["bgp", "database", "dhcp_relay", "lldp", "pmon", "radv", "restapi", "sflow",
-                      "snmp", "swss", "syncd", "teamd", "telemetry"] %}
+                      "snmp", "swss", "syncd", "teamd", "telemetry", "nat"] %}
```
Comment (jleveque) [inline files/build_templates/init_cfg.json.j2]: Can you please place "nat" in alphabetical order amongst the others?

**Comment by akokhan** [comment]
retest vs

File: files/build_templates/init_cfg.json.j2 (hunk 1 above)
Comment (akokhan) [reply files/build_templates/init_cfg.json.j2]: done

**Comment by akokhan** [comment]
retest default please

**Comment by akokhan** [comment]
retest mellanox please

**Comment by akokhan** [comment]
retest vsimage please

**Comment by akokhan** [comment]
retest default please

**Comment by akokhan** [comment]
retest default please

**Comment by akokhan** [comment]
retest vsimage please

**Comment by akokhan** [comment]
retest default please

**Comment by akokhan** [comment]
retest this please

**Comment by akokhan** [comment]
retest vs please

**Comment by akokhan** [comment]
retest vsimage please

**Pull request merged**